apps/examples/wc-next10/ButtonSizes.js

```diff
@@ -21,7 +13,7 @@ export default function ButtonSizes () {
                                 const size= BUTTON_SIZE[key];
 
                                 return (
-                                    <PieBtn size={size} key={key}>{key.toLowerCase()}</PieBtn>
+                                    <PButton size={size} key={key}>{key.toLowerCase()}</PButton>
```

Comment:
More a question than anything else – is this the naming convention we've agreed to go with when importing (or showing the import) of components? Not sure if others have any thoughts on this, but would be good to document this somewhere if we have come to some sort of consensus.

/cc @xander-marjoram @fernandofranca ?

Reply:
@fernandofranca @ashleynolan I've tried using the exports but unfortunately it doesn't work due to how the config is set up. The `createComponent` function is generated in the index.ts under the component instead of in a separate file - I tried generating it in a separate file called `react.ts` but I couldn't seem to get this to work and compile. Placing it under the component means it works easily, but we have to export it as a different name. I've changed it to `Pie(component)React` so it's clear that this is specific for react components. 

Happy to try and get the exports working if anyone wants to pair? 🙏  If not, maybe we can keep it as the React name for now and investigate in a separate ticket 😄